docs/packages/web/README.md

```diff
@@ -283,8 +283,11 @@ module.exports = {
       // Example: Remove console and debugger from output
       minify: {
         babel: {
-          removeConsole: true,
-          removeDebugger: true,
+          minify: {
+            removeConsole: true,
+            removeDebugger: true,
+          },
+          preset: {}
```

Comment:
I think this should be `plugin` rather than `preset`. See:
https://github.com/sammarks/neutrino-dev/tree/03689f6a60e896e173bb8e1f47141a483ecc8a2e/docs/packages/babel-minify#usage
https://github.com/mozilla-neutrino/neutrino-dev/blob/release/v8/packages/babel-minify/index.js#L5

Reply:
@sammarks do you know if you'll have a chance to make the change above, or would you prefer me to update it? :-)